datalake/src/main/java/com/sequenceiq/datalake/flow/create/SdxCreateHandlerSelectors.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.datalake.flow.create;
+
+import com.sequenceiq.flow.core.FlowEvent;
+
+public enum SdxCreateHandlerSelectors implements FlowEvent {
```

Comment:
There is a `SdxCreateEvent` enum for SDX create flow events.

Reply:
Yes, I am aware. It is also being extended in this PR. :smile: That class only collects success/failure events sent from handlers. I created this one to separate the ones that are used for triggering handlers. Ideally, all the existing hardwired selector strings in other SDX handlers (and their request classes) could be extracted here to avoid duplications.

Similar `HandlerSelectors` approaches are also used elsewhere, e.g. in environment and core.

Reply:
File has been removed following the refactors that Adam asked below. :smile: